tests/unit_tests/libs/controllers/test_predictor.py

```diff
@@ -100,16 +100,18 @@ def test_explain_prediction(self):
         mdb.learn(
             from_data=input_dataframe,
             to_predict='numeric_y',
-            stop_training_in_x_seconds=1
+            stop_training_in_x_seconds=15
```

Comment:
Is it required to train for so long? I think this can be as short as possible if the test only tests the interfaces

Reply:
No, it doesn't, I don't remember why I increased it, will reduce this number.